### DISCUSSION TITLE
Awesome Thermostat is saying Hello, Passing by :)

### DISCUSSION THREAD

**Discussion by dadge** [jmcollin78/versatile_thermostat#563] [General]
Hi,
I am the owner of "Awesome thermostat" repository. Just wanted to say that I am honored by your mention in Readme. Also, I am happy that somebody continues my work. 
I must admit that I let it down when I changed my old heaters with Daikin ACs.
Anyways, I an know searching a solution as I just installed a new module for my dryer in my bathroom : nodon sin-4-fp-21.
My uses cases are the following : 
-  map the presets with the nodOn ones (comfort (target temperature = heater set temperature) eco (target temperature = heater set temperature - 3.5°C), frost_protection (target temperature = 7 to 8°C), off (heater stops heating), and the less commonly used comfort_-1 (target temperature = heater set temperature - 1°C), comfort_-2 (target temperature = heater set temperature - 2°C))
- Set presets  to : 
    - comfort when temperature is below target, 
    - eco when when temperature is above target
    - frost_protection when the windows is open 
- turning on and off the fan based on humidity sensor and value (I think generic climate has this feature now).

I will make a try with your implementation. Or make another one, Or fork you If don't manage to fulfill my requirements ;)

Bonne journée !

**Comment by jmcollin78** [comment]
Hello @dadge,

I was very happy to continue your inspiring work. Since 2 years now, many enhancements have been added, but you should see some of your code intact.

I don't have a direct mapping for heater with pilot wire. Instead I encourage people to create a virtual switch a use the `over_switch` VTherm, which is under optimal.

If you want to come back to coding, it would be a great pleasure for me, to help you add this new type of VTherm. Let me know.

And if you speak French, I would be glad to talk in French. My English is so bad...
